packages/docusaurus-plugin-client-redirects/src/redirectCreators.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {RedirectsCreator} from './types';
+import {removeTrailingSlash} from './utils';
+
+export function fromExtensionsRedirectCreator(
+  extensions: string[],
+): RedirectsCreator {
+  const dottedExtensions = extensions.map((ext) => `.${ext}`);
+  return (fromRoutePath: string) => {
+    const extensionMatch = dottedExtensions.find((ext) =>
+      fromRoutePath.endsWith(`.${ext}`),
+    );
+    if (extensionMatch) {
```

Comment:
I would change the `extensionMatch` const name to something more understandable, maybe `foundExtension`?

Reply:
done